tests/algorithms/test_low_precision_groupnorm.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+from torch.nn import LayerNorm
+from torch.utils.data import DataLoader
+
+from composer.algorithms.low_precision_groupnorm import LowPrecisionGroupNorm, apply_low_precision_groupnorm
+from composer.algorithms.low_precision_groupnorm.low_precision_groupnorm import LPGroupNorm
+from composer.core import Event, Precision, State
+from composer.loggers import Logger
+from composer.utils import get_device
+from tests.common import RandomImageDataset
+from tests.common.models import SimpleGroupNormConvModel
+
+
+def assert_is_lpgn_instance(model):
+    # ensure that within the entire model, no PyTorch LayerNorm exists, and at least one APEX FLN does.
+    assert model.modules is not None, 'model has .modules method'
+    for module_class in model.modules():
+        if isinstance(module_class, LayerNorm):
+            assert isinstance(
+                module_class,
+                LPGroupNorm), 'A standard torch.nn.LayerNorm should not be found in the model after surgery is applied.'
```

Comment:
less awkward line splitting?
```suggestion
            message = ('A standard torch.nn.LayerNorm should not ' +
                       'be found in the model after surgery is applied.')
            assert isinstance(module_class, LPGroupNorm), message
```